sky/server/server.py

```diff
@@ -1081,14 +1081,14 @@ async def complete_storage_name(incomplete: str,) -> List[str]:
     # that it is shown only when the API server is started.
     usage_lib.maybe_show_privacy_policy()
 
-    num_workers = None
+    num_workers = 1
     if cmd_args.deploy:
-        num_workers = os.cpu_count()
+        num_workers = common_utils.get_cpu_count()
 
     workers = []
     try:
         workers = executor.start(cmd_args.deploy)
-        logger.info('Starting SkyPilot API server')
+        logger.info(f'Starting SkyPilot API server, workers={num_workers}')
```

Comment:
This might be a bit too detailed for a user to know. Can we hide the num workers in `logger.debug` or some other logs, not directly printing to the terminal?

Reply:
Ahh, that makes sense, as long as it is not shown to the user in console, it is good to me.